### PR TITLE
Add Daniel Gonzalez (gonzolino) as reviewer

### PR DESCRIPTION
**Pull request by gonzolino**

### CODE REVIEW

File: OWNERS [hunk 1]
```diff
@@ -3,8 +3,10 @@ approvers:
 - NickrenREN
 - dims
 - FengyunPan2
+- gonzolino
```
Comment (dims) [inline OWNERS]: Let's start as a reviewer and then we can move to approver after some time. is that ok?

Comment (gonzolino) [reply OWNERS]: Yes of course

**Pull request merged**